HttpResponseImpl.java

```diff
@@ -0,0 +1,43 @@
+import java.io.OutputStream;
+import java.util.Map;
+
+public class HttpResponseImpl implements HttpResponse{
+    private String version;
+    private int StatusCode;
+    private String StatusMessage;
```

Comment:
Igual paso en este codigo, mantener las variables en minuscula, asi mantenemos con la convension de java.
```suggestion
    private int statusCode;
    private String statusMessage;
```